src/Notifier.php

```diff
@@ -12,8 +12,6 @@
 define('ERR_IP_RATE_LIMITED', 'phpbrake: IP is rate limited');
 define('ERR_NOTIFICATIONS_DISABLED', 'phpbrake: error notifications are disabled');
 
-const AIRBRAKE_NOTIFIER_VERSION = '0.8.0';
```

Comment:
Why was this constant removed?

When I run the [unit tests](https://github.com/airbrake/phpbrake#running-tests), I see errors mentioning the removed constant. Example:
```
15) Warning
The data provider specified for Airbrake\Tests\NotifierTest::testPostsToURL is invalid.
Undefined constant "Airbrake\AIRBRAKE_NOTIFIER_VERSION"
```